instrumentation/kafka-clients-0.11/src/main/java/io/opentelemetry/instrumentation/auto/kafkaclients/Base64Decoder.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.auto.kafkaclients;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+// TODO - can be removed when JDK7 is dropped (adapted from JDK8 source)
```

Comment:
Hmm - the words `adapted from JDK8 source` seem to indicate a GPL violation. I wouldn't have noticed if we weren't honest about it ;) But now it's unrevertable.

Maybe we can go ahead and make this package Java 8+ now and use the standard lib, or punt on this commit for now.

Reply:
I recommend entirely removing this, and the feature. It was implemented to help a customer using Kafka mirrormaker, and there were probably better, infrastructural solutions.

Reply:
Yeah, @trask https://github.com/open-telemetry/opentelemetry-java-instrumentation/commit/46682e9b70cfa48b6951300a02ecbe72410eb0dc can be dropped.  I don't think we need it in this project.